tests/components/portainer/test_sensor.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the Portainer sensor platform."""
+
+from unittest.mock import AsyncMock, patch
+
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+async def test_all_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_portainer_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
```

Comment:
see https://docs.astral.sh/ruff/rules/pytest-fixture-param-without-value/

```suggestion
@pytest.mark.usefixtures("mock_portainer_client")
async def test_all_entities(
    hass: HomeAssistant,
    snapshot: SnapshotAssertion,
    mock_config_entry: MockConfigEntry,
    entity_registry: er.EntityRegistry,
) -> None:
```

Reply:
Good insight on convention! Thanks for pointing out, @tr4nt0r!